web/app/js/components/ResourceDetail.jsx

```diff
@@ -217,69 +219,62 @@ export class ResourceDetailBase extends React.Component {
       <div>
         {
           this.state.resourceIsMeshed ? null :
-          <div className="page-section">
+          <React.Fragment>
             <AddResources
               resourceName={this.state.resourceName}
               resourceType={this.state.resourceType} />
-          </div>
+          </React.Fragment>
         }
 
-        <div className="page-section">
+        <React.Fragment>
           <Octopus
             resource={this.state.resourceMetrics[0]}
             neighbors={this.state.neighborMetrics}
             unmeshedSources={_.values(this.state.unmeshedSources)}
             api={this.api} />
-        </div>
+        </React.Fragment>
 
-        {
-          !this.state.resourceIsMeshed ? null :
-          <div className="page-section">
-            <TopModule
-              pathPrefix={this.props.pathPrefix}
-              query={topQuery}
-              startTap={true}
-              updateNeighbors={this.updateNeighborsFromTapData}
-              maxRowsToDisplay={10} />
-          </div>
-        }
+        <TopRoutesTabs
+          query={query}
+          pathPrefix={this.props.pathPrefix}
+          updateNeighbors={this.updateNeighborsFromTapData}
+          disableTop={!this.state.resourceIsMeshed} />
```

Comment:
i don't see `disableTop` used?

Reply:
ah good catch, it is supposed to be used by TopRoutesTabs. fixed!